tests/test_create_tile_map.py

```diff
@@ -60,3 +63,34 @@ def create_test_geotiff(output_file, geotransform, shape, epsg):
     band = dataset.GetRasterBand(1)
     band.WriteArray(np.ones(shape, dtype=int))
     dataset = None
+
+
+@pytest.fixture(autouse=False)
```

Comment:
This is already the default, right? Removing it would make it more clear that it's no different than the other `@pytest.fixture` above.
```suggestion
@pytest.fixture
```